changelog/security/2022-01-07-containerd-1.5.9.md

```diff
@@ -0,0 +1 @@
+- [CVE-2021-43816](https://github.com/advisories/GHSA-mvff-h3cj-wj9c)
```

Comment:
Thanks for adding the security changelog entry. :) 
```suggestion
- [CVE-2021-43816](https://nvd.nist.gov/vuln/detail/CVE-2021-43816)
```

To be consistent with the release notes, we tend to use the NIST as reference.

Reply:
Thanks, I just asked @dongsupark about exactly the same thing